tasks/Task.hpp

```diff
@@ -31,11 +31,16 @@ namespace usbl_evologics {
 
 	    boost::shared_ptr<Driver>  driver;
 
+	    // Queue of Instant Messages to be transmitted to remote device.
 	    std::queue<SendIM> queueSendIM;
```

Comment:
Consistency ... you name this queue queueSendIM but the raw queue is called queueRawPacket. Put "Send" in both or in none.